src/Core/Manifest/Module.php

```diff
@@ -127,9 +128,15 @@ public function unserialize($serialized)
      */
     public function activate()
     {
-        $config = "{$this->path}/_config.php";
-        if (file_exists($config)) {
-            require_once $config;
+        global $project;
+        $config = ModuleManifest::config();
+        $configPath = "{$this->path}/_config.php";
+        if (file_exists($configPath)) {
+            require_once $configPath;
+            if ($project !== $config->get('project')) {
```

Comment:
Risk is that `global $project` is already different to the config project before the _config.php is included. I suggest to assign the global to a temp variable, include _config.php, and then compare back at that point.